common/containers/Tabs/ScheduleTransaction/components/ScheduleFields.scss

```diff
@@ -0,0 +1,43 @@
+@import 'common/sass/variables';
+
+.scheduled-tx-settings {
```

Comment:
We follow the SuitCSS class naming convention, though these class names are very close it'd be nice to have them match the rest of the codebase: https://github.com/MyCryptoHQ/MyCrypto/wiki/Style-Guides#classnames